packages/plugin-core/package.json

```diff
@@ -442,6 +457,12 @@
             }
         ],
         "keybindings": [
+            {
+                "command": "dendron.pasteLink",
+                "key": "shift+ctrl+v",
+                "mac": "shift+cmd+v",
```

Comment:
this conflicts with an existing vscode keyboard shortcut (open markdown preview). You can detect existing keybindings by using the guide [here](https://code.visualstudio.com/docs/getstarted/keybindings#_detecting-keybinding-conflicts). In this case, I wouldn't define a shortcut but rather leave it up to the user